app/models/ability.rb

```diff
@@ -309,10 +309,11 @@ def package_abilities
              search_stockit_items remove_from_set designate register_quantity_change
              mark_missing move print_inventory_label stockit_item_details
              split_package add_remove_item contained_packages parent_containers
-             fetch_added_quantity], Package
+             fetch_added_quantity requested_packages], Package
```

Comment:
- remove the requested_packages from both places
- add it in the requested_packages_abilities as
```ruby
def requested_packages_abilities
    can %i[create destroy index checkout update], RequestedPackage, user_id: @user_id
  end
```

Reply:
👍